avocado/core/sysinfo.py

```diff
@@ -214,8 +215,8 @@ def stop(self):
             process.kill_process_tree(self.pipe.pid)
             retcode = self.pipe.wait()
         else:
-            log.error("Daemon process '%s' (pid %d) terminated abnormally (code %d)",
-                      self.cmd, self.pipe.pid, retcode)
+            log.error("Daemon process '%s' (pid %d) terminated abnormally "
+                      "(code %d)", self.cmd, self.pipe.pid, retcode)
```

Comment:
There are no issues with these line disposition, but, bear in mind that you can go as far as 79 chars, which would result in:

```python
# following line is column numbers
#2345678901234567890123456789012345678901234567890123456789012345678901234567890
                log.error("Daemon process '%s' (pid %d) terminated abnormally "
                          "(code %d)", self.cmd, self.pipe.pid, retcode)
```

Reply:
ah yes i miss that ,will make is pep8 complaint